### DISCUSSION TITLE
Transition and recommendation for compatible svgs?

### DISCUSSION THREAD

**Discussion by NikOcaml** [matteobruni/tsparticles#2355] [Q&A] [answered]
Hi,
first of all - thank you so much Matteo.

I would  like to use the particles dynamically. Is it possible to transition from one svg to another? Combined with a fixed background this would be my dream combo.

Secondly, is there any tool you can recommend for creating a compatible svg? I can't manage to even get a simple "D" made out of particles.

**Comment by matteobruni** [comment, marked as answer]
Hi @NikOcaml,

The Polygon Mask feature has no transition effect, it was thought for static SVG files.

For creating compatible SVG files I use Inkscape, and I save them in `1.1` simple format. Be careful that only `path` tags will be used for the mask, Inkscape can save every shape in `path` tags with the simple `1.1` format, I don't remember which flags should be set but it's not that hard to find them with some tries.

**Reply by NikOcaml** [reply]
Thanks @matteobruni !
Maybe I'll be good enough to contribute it in a couple of months.

I would use a fixed tsparticle.js canvas which transforms into headers or simple logos depending on the scroll position in the document!

BTW: Here's a tool that works for creating compatible svg texts:
https://danmarshall.github.io/google-font-to-svg-path/
